Run Hive Glue test on Travis

rentaow opened the pull request.

rentaow commented on .travis.yml:
```diff
@@ -101,12 +101,16 @@ script:
-      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID ]]; then
-        env AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
-        AWS_SECRET_ACCESS_KEY=$S3_TESTS_AWS_SECRET_ACCESS_KEY \
+      if [[ -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
```
Not sure where these are set in Travis but the same access keys can be reused for Glue.

sopel39 commented on presto-hive/bin/run_hive_glue_tests.sh:
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
I don't think this script is needed as it simply launched maven with selected profile. This can be done within Travis yaml directly.

rentaow replied:
Sure, I will remove this script.

sopel39 commented on .travis.yml:
```diff
@@ -101,12 +101,16 @@ script:
+        AWS_SECRET_ACCESS_KEY=$HIVE_TESTS_AWS_SECRET_ACCESS_KEY \
         S3_BUCKET_ENDPOINT=$S3_TESTS_BUCKET_ENDPOINT \
         S3_BUCKET=$S3_TESTS_BUCKET \
         presto-hive-hadoop2/bin/run_hive_s3_tests.sh
```
If you want to reuse this `if` then add another nested if here that `HIVE_TESTS_AWS_ACCESS_KEY_ID` and `HIVE_TESTS_AWS_SECRET_ACCESS_KEY` are set before launching s3 tests

rentaow replied:
I was intending on reusing the same AWS key for both S3 & Glue access which is why I renamed ```S3_TESTS_AWS_ACCESS_KEY_ID``` to something more generic like ```HIVE_TESTS_AWS_ACCESS_KEY_ID```.  

I wasn't sure how or where this key is set within Travis but anyways, using ```S3_TESTS_AWS_ACCESS_KEY_ID``` can work as well for accessing Glue as long as the AWS user for that key has permission to Glue APIs.

sopel39 commented on .travis.yml:
```diff
@@ -101,12 +101,16 @@ script:
-      if [[ -v S3_TESTS_AWS_ACCESS_KEY_ID ]]; then
-        env AWS_ACCESS_KEY_ID=$S3_TESTS_AWS_ACCESS_KEY_ID \
-        AWS_SECRET_ACCESS_KEY=$S3_TESTS_AWS_SECRET_ACCESS_KEY \
+      if [[ -v HIVE_TESTS_AWS_ACCESS_KEY_ID ]]; then
```
should we different key for glue tests? We internally run S3 tests. Would the same key work for glue test even though we didn't setup anything specifically for glue?

sopel39 replied:
sounds good. We can stick with single key then? Are glue API calls enabled by default?

rentaow replied:
> Are glue API calls enabled by default?
Not sure, you would have to check the IAM policies for that access key.

rentaow commented:
Addressed comments. I kept rename of ``` S3_TESTS_AWS_ACCESS_KEY_ID``` -> ```HIVE_TESTS_AWS_ACCESS_KEY_ID```, let me know if you don't want this to be renamed.
